--- a/bigtop-packages/src/common/spark/do-component-build	2017-06-01 16:21:26.000000000 -0500
+++ b/bigtop-packages/src/common/spark/do-component-build	2017-09-07 18:50:51.000000000 -0500
@@ -16,6 +16,11 @@

 set -ex

+# Snap builders create a proxy-aware m2 settings file; use it if present
+if [ -f $SNAPCRAFT_PART_INSTALL/../m2/settings.xml ] ; then
+  mkdir -p $HOME/.m2 && ln -s $SNAPCRAFT_PART_INSTALL/../m2/settings.xml $HOME/.m2/settings.xml
+fi
+
 . `dirname $0`/bigtop.bom

 # BIGTOP-2607
